rosdep/base.yaml

```diff
@@ -404,6 +404,7 @@ clang-format:
   ubuntu: [clang-format]
 clang-tidy:
   debian:
+    bionic: [clang-tidy]
```

Comment:
Bionic is an Ubuntu distribution, not a Debian one.  Also, bionic is already covered by the wildcard on the ubuntu keys.  So I'm going to close this out.

Reply:
In Ubuntu 18.04.4 for Raspberry Pi 4, it says that there is no package candidate for clang-tidy.

Reply:
[clang-tidy](https://packages.ubuntu.com/bionic/clang-tidy) is listed as being present for arm64 and armhf. It sounds like your Pi may not be configured with the upstream Ubuntu repositories.